test/runtests.jl

```diff
@@ -45,3 +45,13 @@ result = zeros(Float64, 5)
     Libxc.xc_func_end(ptr)
     Libxc.xc_func_free(ptr)
 end
+
+@testset "High-level interface" begin
+    func = Libxc.Functional(:lda_x)
+    Libxc.evaluate_lda!(func, rho, E=result)
```

Comment:
I think result need to be reset to zeros to avoid confusion.

Reply:
Good point. Changed now.